solutions/CF Kayaking.mdx

```diff
@@ -0,0 +1,52 @@
+---
+id: kayaking

```

Comment:
I think is better to put kayaking-CF or something like that, since there could be more problems with the same name.